### PR TITLE
Make categories always present.

### LINKED ISSUE #207: Upvote/save INSIDE the item card
With the inside having a cancel/close card button instead of the list of categories:

Cancel ... ... ... ... Saved Vote

### PR DESCRIPTION
**Pull request by alexgerstein**
Closes #207.

### CODE REVIEW

**Comment by swglad** [comment]
Cards are getting "knocked out of place" in their row and leaving empty spaces. Is there a way to either make both cards in each row the same length (like before) or have the cards be offset from one another and not always line up (like notey: http://www.notey.com/blogs/outfit)?

screen shot 2015-05-03 at 6 00 32 pm

File: sourcemash/frontend/templates/item-card.tpl [hunk 1]
```diff
@@ -43,4 +24,22 @@
       <p class="flow-text summary"><%= item.get('summary')%></p>
     </div>
   </div>
+
+  <div class="card-action z-depth-1 valign-wrapper row">
+    <div class="category-list flow-text col s8">
+      <% _.each(categories, function(category) { %>
```
Comment (swglad) [inline sourcemash/frontend/templates/item-card.tpl]: Only categories N > 1

Comment (alexgerstein) [inline sourcemash/frontend/templates/item-card.tpl]: Fixed.

**Comment by alexgerstein** [comment]
The offset should be fixed now that the categories > 1 should be resolved.

**Pull request merged**